server/data/data-file/src/main/java/org/apache/james/mailrepository/file/FileMailRepository.java

```diff
@@ -244,7 +244,7 @@ public Mail retrieve(String key) throws MessagingException {
 
             return mc;
         } catch (Exception me) {
-            LOGGER.error("Exception retrieving mail: " + me);
+            LOGGER.error("Exception retrieving mail: {}", me.toString());
```

Comment:
LOGGER.error("Exception retrieving mail", me);

Overwize we swallow the stacktrace

Reply:
Done